domains/authorization/Authorization.API/API.IntegrationTests/API.IntegrationTests/API.IntegrationTests.csproj

```diff
@@ -6,18 +6,32 @@
         <Nullable>enable</Nullable>
 
         <IsPackable>false</IsPackable>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
         <IsTestProject>true</IsTestProject>
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="coverlet.collector" Version="6.0.0"/>
+        <PackageReference Include="FluentAssertions" Version="6.12.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.4" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="8.0.4" />
```

Comment:
Are we still using inmemory db?

Reply:
Removed:
https://github.com/Energinet-DataHub/energy-origin/pull/1059/commits/c88ce25bc88103d8216ab9c66df7a3a33db6cbe6